packages/effect/test/RateLimiter.test.ts

```diff
@@ -1,223 +1,271 @@
-import { Clock, Deferred, Effect, Either, Fiber, pipe, Ref, TestClock } from "effect"
+import { Clock, Deferred, Effect, Either, Fiber, Option, RateLimiter, ReadonlyArray, Ref, TestClock } from "effect"
 import * as it from "effect-test/utils/extend"
-import { none } from "effect/Option"
-import * as RateLimiter from "effect/RateLimiter"
 import { assert, describe } from "vitest"
 
-describe("RateLimiterSpec", () => {
-  it.effect("execute up to max calls immediately", () => {
+describe("RateLimiter", () => {
+  describe("fixed-window", () => {
+    RateLimiterTestSuite("fixed-window")
+
+    it.effect("uses the token-bucket algorithm for token replenishment", () =>
```

Comment:
Should move this to the token bucket block.

Reply:
That was dumb - thanks 👍